test/e2e/dra/dra.go

```diff
@@ -105,6 +110,49 @@ var _ = ginkgo.Describe("[sig-node] DRA [Feature:DynamicResourceAllocation]", fu
 	})
 })
 
+// getExternalClaimTest generates test function for the external claim for certain allocation mode
+func genExternalClaimTest(ctx context.Context, f *framework.Framework, driver *Driver, allocationMode corev1.AllocationMode) func() {
+	return func() {
+		var b = newBuilder(f, driver)
+
+		ginkgo.It("registers plugin", func() {
+			// If we got here, the driver is running.
+		})
+		ginkgo.It("supports simple pod referencing external resource claim", func() {
+			parameters := b.resourceClaimParameters()
+			pod := b.podExternal()
+			b.create(ctx, parameters, b.externalClaim(allocationMode), pod)
```

Comment:
Let's stay consistent and have all of the create calls on a separate line.

Reply:
This wasn't resolved?

I've changed it in https://github.com/bart0sh/kubernetes/pull/4/files

Reply:
wasn't it?